pybamm/__init__.py

```diff
@@ -12,37 +12,11 @@
 #
 # Version info
 #
-def _load_version_int():
-    try:
-        root = os.path.abspath(os.path.dirname(__file__))
-        with open(os.path.join(root, "version"), "r") as f:
-            version = f.read().strip().split(",")
-        year, month = [int(x) for x in version]
-        return year, month
-    except Exception as e:
-        raise RuntimeError("Unable to read version number (" + str(e) + ").")
+from pybamm.version import __version__
 
-
-__version_int__ = _load_version_int()
-__version__ = ".".join(["{:02d}".format(x) for x in __version_int__])
 if sys.version_info[0] < 3:
     del x  # Before Python3, list comprehension iterators leaked
```

Comment:
We should remove these as we no longer support Python 2

Reply:
Yes